### PR TITLE
feat/husky

### PR DESCRIPTION
**Pull request by ottonotto**
Pushing issue is now resolved.

### CODE REVIEW

**Comment by shams-ali** [comment]
husky is a feature not documentation....

CONTRIBUTING.md was documentation

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
-
+.env  
+package-lock.json
```
Comment (shams-ali) [inline .gitignore:3]: this should not be ignored

File: src/reportWebVitals.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-  if (onPerfEntry && onPerfEntry instanceof Function) {
-    import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
-      getCLS(onPerfEntry);
-      getFID(onPerfEntry);
-      getFCP(onPerfEntry);
-      getLCP(onPerfEntry);
-      getTTFB(onPerfEntry);
-    });
-  }
+const reportWebVitals = (onPerfEntry) => {
```
Comment (shams-ali) [inline src/reportWebVitals.js:1]: these parens shouldnt be needed if theres only one param...please fix the eslint rule for this

File: .eslintignore [hunk 3]
```diff
@@ -0,0 +1 @@
+**/node_modules
```
Comment (shams-ali) [inline .eslintignore:1]: end of line

File: src/reportWebVitals.js (hunk 2 above)
Comment (ottonotto) [reply src/reportWebVitals.js]: fixed

File: .eslintignore (hunk 3 above)
Comment (ottonotto) [reply .eslintignore]: fixed

File: .gitignore (hunk 1 above)
Comment (ottonotto) [reply .gitignore]: git-it

**Pull request merged**